SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
 # Es zeigt die Rohstoffquellen (Gold, Holz, usw.) im 123,456X Format. X ist eine Zehnerpotenz (z.B. 123,456,789 wird sich als 123,456K zeigen)
 # Εμφανίζει τους φυσικούς πόρους (χρυσό, ξύλο κλπ.) χρησιμοποιώντας τη μορφή 123,456Χ, όπου Χ είναι μια δύναμη του δέκα (π.χ. το 123,456,789 θα εμφανίζεται ως 123,456K)
 # Pokazuje ilość surowców (złoto, drewno, itp.) używając formatowania 123,456X , gdzie X jest odpowiednim przedrostkiem liczby 10 (np. ilość 123,456,789 zostanie wyświetlona jako 123,456K).
-ResourcesFormat = 1
+ResourcesFormat = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.